ui/src/seminars/components/SeminarItem.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+
+import { VideoCameraAddOutlined } from '@ant-design/icons';
+import { Row, Col } from 'antd';
+import EditRecordAction from '../../common/components/EditRecordAction';
+import ResultItem from '../../common/components/ResultItem';
+import { SEMINARS } from '../../common/routes';
+import Title from '../../common/components/Title';
+import WebsitesAction from '../../common/components/WebsitesAction';
+import IconText from '../../common/components/IconText';
+import AuthorList from '../../common/components/AuthorList';
+import SeminarDatesAndTimes from './SeminarDatesAndTimes';
+import {
+  getLocalTimezone,
+  doTimezonesHaveDifferentTimes,
+} from '../../common/utils';
+
+function SeminarItem({ metadata, selectedTimezone }) {
+  const renderDatesInSelectedAndLocalTimezone = useCallback(
```

Comment:
doesn't need to be a function can go directly into the markup, since it is just used once.